3-UsingAPIs/Week1/homework/ex2-checkDoubleDigits.js

```diff
@@ -10,10 +10,16 @@ Complete the function called `checkDoubleDigits` such that:
   "Expected a double digit number but got `number`", where `number` is the 
   number that was passed as an argument.
 ------------------------------------------------------------------------------*/
-function checkDoubleDigits(/* TODO add parameter(s) here */) {
+function checkDoubleDigits(number) {
   // TODO complete this function
+  return new Promise((resolve, reject) => {
+    if (number > 10 && number < 99) {
```

Comment:
Something I overlooked the first time: number 10 and 99 themselves are also double digit.

Reply:
Yes you  are right, i changed them with 9 and 100.